Source/Fuse.Controls.Panels/Panel.Freeze.uno

```diff
@@ -226,7 +226,7 @@ namespace Fuse.Controls
 				Texture: buffer.ColorBuffer;
 				Invert: true;
 				
-				PixelColor: float4( prev.XYZ, prev.W * Opacity );
+				PixelColor: prev * Opacity;
```

Comment:
This converts to premultiplied output, doesn't it? Was that intended?

Reply:
It doesn't convert, it multiplies with the alpha according to pre-multiplied rules. The image is rendered on the GPU, and the draw-block uses `Fuse.Drawing.PreMultipliedAlphaCompositing`, we must keep the alpha fully premultiplied.